webapp/app/forms/fields.py

```diff
@@ -19,6 +19,44 @@ def pre_validate(self, form):
         ValidElsterCharacterSet().__call__(form, self)
 
 
+class UnlockCodeWidget(TextInput):
```

Comment:
If we would use the class attributes to make it more adaptable, we should probably call this `SplitFieldWidget` and let `UnlockCodeWidget` inherit from that?

Reply:
Mh, yes. However, I think this field is pretty special in that regard. Here we need three same-sized inputs and yes, it'd make sense to have the input length as a class variable. It gets more complicated though once we take fields into account that have differently-sized input fields? (e.g. date and idnr)
Let's maybe talk about it, I don't quite see the benefit of it yet.